### PR TITLE
CVE-2004-0809

### PR DESCRIPTION
**Pull request by Jonathan-Eid**

### CODE REVIEW

**Comment by Jonathan-Eid** [comment]
I think the master branch pull request was mistake, this is what I meant to merge in. Can you see my finished yml file?

**Comment by yerngpaii** [comment]
continue of #60

File: cves/CVE-2004-0809.yml [hunk 1]
```diff
@@ -32,7 +32,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    Once the WebDAV undergoes an indirect lock, the wrong memory address is accessed and it results in a segmentation
```
Comment (yerngpaii) [inline cves/CVE-2004-0809.yml:36]: What is WebDAV and how does it relate to the HTTPD ?

File: cves/CVE-2004-0809.yml [hunk 2]
```diff
@@ -95,8 +98,8 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: There isn't a lot of information but the vulnerability was found by a consistant maintainer of the project.
```
Comment (yerngpaii) [inline cves/CVE-2004-0809.yml:101]: consistent *

Comment (yerngpaii) [reply cves/CVE-2004-0809.yml:101]: Who is this consistent maintainer ?

File: cves/CVE-2004-0809.yml [hunk 3]
```diff
@@ -108,8 +111,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
-  name:
+  answer: An extension of HTTPD that handles collaboratively editting files on web servers, specifically the locking part
```
Comment (yerngpaii) [inline cves/CVE-2004-0809.yml:114]: collaborative * editing * [of]

File: cves/CVE-2004-0809.yml [hunk 4]
```diff
@@ -119,8 +122,8 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
+    - commit: 32987a707bea265cf56cbe309275f357a873991c
+      note:  It seems like the way locking was designed in this system has been very problamatic for quite some time
```
Comment (yerngpaii) [inline cves/CVE-2004-0809.yml:126]: problematic *

**Review by yerngpaii — CHANGES_REQUESTED** [review]

File: cves/CVE-2004-0809.yml [hunk 5]
```diff
@@ -3,15 +3,15 @@ CWE_instructions: |
...
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
```
Comment (JoshVerDow) [inline cves/CVE-2004-0809.yml:17]: Announced date should be in YYYY-MM-DD format for readability (remove T04:00Z).

File: cves/CVE-2004-0809.yml (hunk 4 above)
Comment (JoshVerDow) [inline cves/CVE-2004-0809.yml:126]: The note for this interesting commit does not really illustrate why it was interesting.  A little more explanation as to the contents of the commit would help for ease of information access.

File: cves/CVE-2004-0809.yml [hunk 6]
```diff
@@ -200,4 +203,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+      The fix presented did make it seem like a coding mistake as I interpret their was a typo causing the wrong memory address to be altered. But looking further in the commits
```
Comment (JoshVerDow) [inline cves/CVE-2004-0809.yml:207]: This first sentence is kind of hard to read.  I would suggest, "The fix presented made it seem like the problem was a coding mistake.  There was a typo causing the wrong memory address to be altered."

File: cves/CVE-2004-0809.yml [hunk 7]
```diff
@@ -200,4 +203,6 @@ mistakes:
...
+      I realize the whole way the system was designed had a lot of faults so there definitely was some design flaws with the way locking worked.
```
Comment (JoshVerDow) [inline cves/CVE-2004-0809.yml:208]: If you found more examples of design faults, maybe add an example for reference?

File: cves/CVE-2004-0809.yml [hunk 8]
```diff
@@ -32,7 +32,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    Once the WebDAV undergoes an indirect lock, the wrong memory address is accessed and it results in a segmentation
+    fault that could cause a system crash.
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
```
Comment (JoshVerDow) [inline cves/CVE-2004-0809.yml:40]: I'd like to add 2 upvotes.

**Pull request merged**